mbedtls/src/ssl/async_io.rs

```diff
@@ -250,4 +318,42 @@ where
     }
 }
 
-// TODO: AsyncIo impl for tokio::net::UdpSocket
+/// A wrapper of [`tokio::net::UdpSocket`] on which
+/// [`tokio::net::UdpSocket::connect`] was successfully called.
+///
+/// Construct this type using [`ConnectedAsyncUdpSocket::connect`].
+pub struct ConnectedAsyncUdpSocket {
+    socket: UdpSocket,
+}
+
+impl ConnectedAsyncUdpSocket {
+    pub async fn connect<A: tokio::net::ToSocketAddrs>(socket: UdpSocket, addr: A) -> StdResult<Self, (IoError, UdpSocket)> {
+        match socket.connect(addr).await {
+            Ok(_) => Ok(ConnectedAsyncUdpSocket { socket }),
```

Comment:
Small nit, but for readability, I prefer to see ``Ok(())`` rather than ``Ok(_)`` because the former doesn't make me look up the signature.

Reply:
I agree. will fix it